src/components/Header/styled.js

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components'
+
+const HeaderImage = styled.img`
+  margin-right: 100px;
+  width: 30px;
+  height: 30px;
+`
+
+const NavListItem = styled.li`
+  margin-left: 20px;
+  padding-bottom: 10px;
+`
+
+const LogoutText = styled.h5`
+  color: white;
+  cursor: pointer;
+`
+
+const LogoutDiv = styled.div`
+  float: left;
```

Comment:
Avoid using `float`, instead you need to look at `flex-box` first and then even better option of `css-grid`

Reply:
Also, with `float` remember to `clear` the container `div` element.